packages/contentstack-migration/examples/taxonomies/import-taxonomies.js

```diff
@@ -0,0 +1,150 @@
+const fs = require('fs');
+const fastcsv = require('fast-csv');
+const find = require('lodash/find');
+
+module.exports = async ({ migration, stackSDKInstance, managementAPIClient, config }) => {
```

Comment:
Is this function expected to be asynchronous? As I don't see any await function!

Reply:
@antonyagustine removed. Correct, we don't need async here; I kept it async because I was following an older script file.